lighthouse-core/computed/page-dependency-graph.js

```diff
@@ -269,8 +269,11 @@ class PageDependencyGraph {
     const networkNodeOutput = PageDependencyGraph.getNetworkNodeOutput(networkRecords);
     const cpuNodes = PageDependencyGraph.getCPUNodes(traceOfTab);
 
-    const rootRequest = networkRecords.reduce((min, r) => (min.startTime < r.startTime ? min : r));
+    // The root request is the earliest network request, using position in networkRecords array to break ties.
+    const rootRequest = networkRecords.reduce((min, r) => (r.startTime < min.startTime ? r : min));
```

Comment:
might be overkill, but should we pull this out into another method on `NetworkAnalyzer` like `findMainDocument`? I can see it usable by others (if they remember it's there).

Reply:
I'd want to shift consumers to be using the graph as the source of truth, so *hopefully* no one else needs to compute this too? :)